generic_config_updater/gu_common.py

```diff
@@ -133,7 +138,7 @@ def validate_config_db_config(self, config_db_as_json):
 
         return True, None
 
-    def validate_lanes(self, config_db):
+    def validate_lanes(self, config_db, dup_lanes_platforms=[]):
```

Comment:
>[] [](http://example.com/codeflow?start=60&length=2)

This is a common bug. ref: [https://stackoverflow.com/a/44272023/2514803](https://stackoverflow.com/a/44272023/2514803)